scrunch/tests/test_expressions.py

```diff
@@ -1590,6 +1623,125 @@ def test_transform_alias_to_var_id(self):
             ]
         }
 
+    def test_process_any_all_exprs(self):
+        var_id = '0001'
+        var_alias = 'MyMrVar'
+        var_type = 'multiple_response'
+        var_url = '%svariables/%s/' % (self.ds_url, var_id)
+
+        table_mock = mock.MagicMock(metadata={
+            var_id: {
+                'id': var_id,
+                'alias': var_alias,
+                'type': var_type,
+                "subvariables": [
+                    "%ssubvariables/001/" % var_url,
+                    "%ssubvariables/002/" % var_url,
+                    "%ssubvariables/003/" % var_url,
+                ],
+                "subreferences": {
+                    "%ssubvariables/001/" % var_url: {"alias": "subvar1"},
+                    "%ssubvariables/002/" % var_url: {"alias": "subvar2"},
+                    "%ssubvariables/003/" % var_url: {"alias": "subvar3"},
+                },
+                "categories": [
+                    {"id": 1, "name": "cat1"},
+                    {"id": 2, "name": "cat2"},
+                    {"id": 3, "name": "cat3"},
+                ]
+            }
+        })
+        ds = mock.MagicMock()
+        ds.self = self.ds_url
+        ds.follow.return_value = table_mock
+        expr = "MyMrVar.any([1])"
+        processed_zcl_expr = process_expr(parse_expr(expr), ds)
+        assert processed_zcl_expr == {
+            'function': 'or',
+            'args': [{
+                'function': 'in',
+                'args': [
+                    {'variable': "%ssubvariables/001/" % var_url},
+                    {'value': [1]}
+                ],
+            }, {
+                'function': 'or',
+                'args': [{
+                    'function': 'in',
+                    'args': [
+                        {'variable': "%ssubvariables/002/" % var_url},
+                        {'value': [1]}
+                    ],
+                },
+                {
+                    'function': 'in',
+                    'args': [
+                        {'variable': "%ssubvariables/003/" % var_url},
+                        {'value': [1]}
+                    ],
+                }],
+            }],
+        }
```

Comment:
I'm still trying to understand what any and all do. But this test feels wrong to me. So here we have 
`(var0001 has value in [1]) or (var0002 has value in [1] or var0003 has value in [1])`
I would expect this to be
``(var0001 has value in [1]) or (var0002 has value in [1]) or (var0003 has value in [1])``
What do you think?